examples/alternate-descriptors/README_es.md

```diff
@@ -0,0 +1,162 @@
+indice-grupo=Técnicas de prueba
+tipo=página
+estado=publicado
+título=Descriptores alternos
+~~~~~~
+
+Véase  [Descriptores alternos](../../alternate-descriptors.html) la página para todos los detalles de como funciona esta característica.
```

Comment:
Since the translated readme file are processed, the final path structure is for example:
`http://localhost:8080/master/es/examples/alternate-descriptors.html`

So the `/es/` path add one extra level that the english version doesn't have:
`http://localhost:8080/master/examples/alternate-descriptors.html`

I propose to change this line to:

```
Véase  [Descriptores alternos](../../../alternate-descriptors.html) la página para todos los detalles de como funciona esta característica.
```